gridstatus/nyiso.py

```diff
@@ -377,7 +386,35 @@ def get_interconnection_queue(self, verbose=False):
             and "SGIA Tender Date" not in completed.columns
         ):
             active = active.drop(columns=["SGIA Tender Date"])
-        completed.columns = active.columns
+        completed_colnames_map = {
```

Comment:
The original code here assumed that the columns were conceptually the same between `active` and `columns`.

After taking a look I saw two differences:

1. There were three `"Proposed COD.X"` columns in `completed` that were not in `active`.
2. The order of `"Proposed Initial-Sync Date"` and `"Proposed In-Service Date"` were flipped between the two.

So I made the mapping explicit.

Reply:
Yeah, it sounds like others have noticed issues between the active sheet and other sheets so thanks for taking care of this (#323).